src/db/repository/postgres/CommunityPostsRepository.ts

```diff
@@ -0,0 +1,34 @@
+import { Kysely } from 'kysely';
+
+import { DB, CommunityPosts } from '../../types/generated/database';
+
+export type CommunityPostsCreateParams = Omit<CommunityPosts, 'id' | 'created_at'>;
+
+export default class CommunityPostsRepository {
+	constructor(private readonly database: Kysely<DB>) {}
+
+	getAll() {
+		return this.database
+			.selectFrom('community_posts')
+			.innerJoin('agencies', 'community_posts.id', 'agencies.id')
+			.selectAll('community_posts')
+			.execute();
+	}
```

Comment:
```suggestion
	getAll() {
		return this.database
			.selectFrom('community_posts')
			.selectAll('community_posts')
			.execute();
	}
```

If we aren't returning anything from the `agencies` table, do we need to still perform the join?